src/redux/actions/UserActions.js

```diff
@@ -0,0 +1,8 @@
+export const UPDATE_USER = 'UPDATE_USER';
```

Comment:
I don't think any of these redux files are needed for this PR. Most of it is going to be overwritten by the authentication PR